tests/MiniProfiler.Tests/Storage/SqlServerStorageTest.cs

```diff
@@ -16,6 +16,7 @@ public class SqlServerStorageTest : BaseTest, IClassFixture<SqlCeStorageFixture<
         
         public SqlServerStorageTest(SqlCeStorageFixture<SqlServerStorageTest> fixture)
         {
+            MiniProfiler.Settings.ProfilerProvider = new WebRequestProfilerProvider();
```

Comment:
the test should be using WebRequestProfilerProvider the DefaultProfilerProvider doesn't save the stopped miniprofiler instances and fails these tests.